### ISSUE TITLE
Fields should be managed to have default data role

### ISSUE THREAD

**Issue by baspeti** [Microsoft/PowerBI-visuals#297]
While trying to develop a map based custom visual, we have found no way to "pair" a data field with any given data role (property).

By pairing I mean, that a given data role could be set as default for a selected data field. For example, when we have `longitude` and `latitude` data fields, we would like to achieve the following behaviour:
When drag 'n' dropping `longitude` and `latitude` fields to the custom visual element itself, they should end up in `longitude` and `latitude` data roles, respectively.

Currently, we have found no way to "bind" data fields to data roles that could result in the above mentioned behaviour.

Here are screenshots with a different Power BI visual that could help to understand our need:
![image](https://user-images.githubusercontent.com/5700926/35152301-2f48b95c-fd22-11e7-83f2-7ff501b1b647.png)

And after releasing the mouse button, it should look like this:
![image](https://user-images.githubusercontent.com/5700926/35152340-56bfab26-fd22-11e7-9561-991376f3af99.png)

So in a custom visual we haven't found any way to control these kind of events.

And here is an already existing issue seems to be connected, but I'm not sure about its status:
https://github.com/Microsoft/PowerBI-visuals/issues/21

**Comment by baspeti** [comment]
And an addition to this issue or request, that for example the built in `Map` visualization almost works as expected, but for some reason it performs an aggregation function, for example currently calculating average, if drag 'n' dropping `latitude` to the visual:

![image](https://user-images.githubusercontent.com/5700926/35152916-9e596c9a-fd24-11e7-86d6-4f0da67ab043.png)

It would be nice, if that aggregation function would not be performed instantly.

**Comment by uve** [comment]
@baspeti, 
You can use [preferredTypes](https://github.com/Microsoft/PowerBI-visuals/pull/282/files#diff-c9034b4b5e89221f7cf3729719e89a74R33) property in capabilities.json
We plan to update the documentation soon.

**Comment by baspeti** [comment]
@uve , thanks for the hints, it worked pretty much. Now it behaves just as other built-in `Map` visualizations.

Now I started to wonder about another thing... as you can see in my last screenshot, in the `Map` visualization when I just simply tick the box next to the Latitude (or Longitude) data field, it lands in the Latitude (Longitude) data role, but as `Average of Latitude`. Why is it happening? Is there a way for us to specify that the default aggregation for Longitude and Latitude roles should be `Don't Summarize`?

**Comment by uve** [comment]
@baspeti, yes, it's known issue and it's in backlog, but there is no ETA for now.
